pymongo/change_stream.py

```diff
@@ -246,9 +258,10 @@ def _run_aggregation_cmd(self, session, explicit_session):
 
     def _create_cursor(self):
         with self._client._tmp_session(self._session, close=False) as s:
+            assert s
```

Comment:
This assert can fail in practice, let's remove it.

Reply:
Done.